src/main/java/mate/academy/service/OrderServiceImpl.java

```diff
@@ -4,11 +4,16 @@
 import java.util.List;
 import mate.academy.model.Order;
 import mate.academy.model.Product;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
 
 public class OrderServiceImpl implements OrderService {
+    private static final Logger logger = LogManager.getLogger(OrderServiceImpl.class);
+
     @Override
     public Order completeOrder(Long userId) {
         // TODO: add log message about method completeOrder was called
+        logger.info("The complete order method was called. Params: userId = {}", userId);
```

Comment:
The logging message 'The complete order method was called. Params: userId = {}' is informative, but consider whether logging user IDs might pose a security risk. If so, you might want to log a more generic message or ensure that user IDs are not sensitive information.